pylinalg/func/misc.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+
+
+def aabb_to_sphere(aabb, /, *, out=None, dtype=None):
+    """A sphere that envelops an Axis-Aligned Bounding Box.
+
+    Parameters
+    ----------
+    aabb : ndarray, [2, 3]
+        The axis-aligned bounding box.
+    out : ndarray, optional
+        A location into which the result is stored. If provided, it
+        must have a shape that the inputs broadcast to. If not provided or
+        None, a freshly-allocated array is returned. A tuple must have
+        length equal to the number of outputs.
+    dtype : data-type, optional
+        Overrides the data type of the result.
+
+    Returns
+    -------
+    sphere : ndarray, [4]
+        A sphere (x, y, z, radius).
+
+    """
+
+    raise NotImplementedError()
+
+
+def abb_transform(aabb, homogenious_matrix, /, *, out=None, dtype=None):
```

Comment:
@Korijn This one seems a little redundant as well, shall we cull it similarly to `vector_project`?

Reply:
This one is not redundant. If the transform contains a rotation element, you cannot simply transform the two points expressing the extrema of the aabb. Instead, the method we employ [in pygfx](https://github.com/pygfx/pygfx/blob/f488647ac9a9f81f3e35ab0d72865a80d9308030/pygfx/linalg/utils.py#L36-L40) is to generate and transform all 8 corners of the aabb, and derive new extrema from them.

Reply:
Ah, so the coordinate frame in which the bounding box is described doesn't transform together with the bounding box?

Then this function indeed has a lot of meaning. I'll update the docstring 👍